packages/perseus/src/widgets/expression/expression.tsx

```diff
@@ -228,34 +200,6 @@ export class Expression
         }
     };
 
-    simpleValidate: (
-        rubric: PerseusExpressionRubric & {scoring?: boolean},
-    ) => PerseusScore = ({scoring, ...rubric}) => {
-        const score = expressionValidator(
-            this.getUserInput(),
-            rubric,
-            this.context.strings,
-            this.context.locale,
-        );
-
-        // "scoring" is a flag that indicates when we are checking answers.
-        // otherwise, we may just be checking validity after changes.
-        if (scoring && score.type !== "invalid") {
```

Comment:
Just to clarify, this means we'll lose this analytics event, right?  I'm not sure that's the worst thing as I don't know that we look at the analytics for this event ever.

Reply:
Yeah, I was going to ask you about that. I don't think it's being used, but if it is being used...it can't be here anyway because `simpleValidate` is going away.

Reply:
Given that a portion of these analytics were to compare usages of the different keypads, I think it's ok to lost this for now.  Perhaps it would fit in `getUserInput`? But I'm not sure how much that function is called (it might be more than once per "answer check").